lib/translator.js

```diff
@@ -0,0 +1,22 @@
+var get = require('lodash/get');
+var escapeRegExp = require('lodash/escapeRegExp');
+var replace = require('lodash/replace');
+var fallbackLocals = require('./translations/en');
+
+function translate(message, params, localization) {
+    if (typeof localization === 'undefined') {
+        localization = fallbackLocals;
+    }
+
```

Comment:
Could your clarify this one a bit? So far as I can see there isnt a `trans` variable? Or do you wanna change `translator` to `translated`?